composer.json

```diff
@@ -3,21 +3,29 @@
 	"description": "Provides email notifications for visitor comments",
 	"type": "silverstripe-module",
 	"license": "BSD-3-Clause",
-	"authors": [
-		{
-			"name": "Damian Mooyman",
-			"email": "damian@silverstripe.com"
-		}
-	],
+	"authors": [{
+		"name": "Damian Mooyman",
+		"email": "damian@silverstripe.com"
+	}],
 	"require": {
-		"silverstripe/comments": "^2.0"
+		"silverstripe/comments": "^3.0"
 	},
 	"require-dev": {
-		"phpunit/PHPUnit": "~3.7@stable"
-	},
+        "phpunit/PHPUnit": "^5.7"
+    },
+    "scripts": {
+        "lint": "phpcs src/ tests/",
+        "fix": "php-cs-fixer fix src/ && php-cs-fixer fix tests/"
+    },
```

Comment:
we probably don't need to add these, or if we do they should reference a local dependency

Reply:
So that's just what comments has so it's consistent.